src/legacy/ui/public/saved_objects/saved_objects_client.ts

```diff
@@ -334,18 +332,17 @@ export class SavedObjectsClient {
       version,
     };
 
-    const request: Promise<UpdateResponse<T>> = this.request({
+    return this.request({
```

Comment:
is there a reason to use promises instead of async/await?

Reply:
I suspect this was written before async/await support but I don't think the code will be that much easier to read.